src/features/pilot_management/PilotSheet/components/StatblockDialog.vue

```diff
@@ -32,15 +33,19 @@ export default class StatblockDialog extends Vue {
   readonly pilot: Pilot
 
   mechSelect = ""
-  buildSummary = false
   discordEmoji = false
+  genRadios = 'mechBuild'
 
   get statblock(): string {
+    if (this.pilot.Mechs.length > 0) {
+      this.mechSelect = this.pilot.ActiveMech.ID ?? this.pilot.Mechs[this.pilot.Mechs.length-1].ID ?? ''
```

Comment:
I think you want to initialize `mechSelect` as part of `mounted` or `created` instead of what you have here? Then have the dropdown select re-assign the value of mechSelect